jax/_src/state/types.py

```diff
@@ -94,7 +92,7 @@ def at(self):
 
 
 # We need an aval for `Ref`s so we can represent `get` and `swap` in Jaxprs.
-class AbstractRef(core.AbstractValue, Generic[Aval]):
+class AbstractRef(core.AbstractValue):
```

Comment:
This generic parameter seems to be completely unused both within the class and in users.

Reply:
I would ask @sharadmv about this, it may be used somewhere outside the JAX package.

Reply:
Okay, but did you notice that the generic parameter doesn't appear in the class at all?  In other words, it doesn't do anything.